### ISSUE TITLE
Loss behavior on Vanilla DINO

### ISSUE THREAD

**Issue by fatemeh-slh** [facebookresearch/dino#76]
Hi,

Thanks again for sharing the code.
I have tried to reproduce the Vanilla DINO results on 8 GPUs using fp16 for 100 epochs. I've used the same args as in [args](https://dl.fbaipublicfiles.com/dino/dino_deitsmall16_pretrain/args.txt) except for the number of epochs. However, the loss values are pretty different from [yours](https://dl.fbaipublicfiles.com/dino/example_runs_logs/dino_vanilla_deitsmall16_log.txt). 

My log:
```
{"train_loss": 7.698364663490955, "train_lr": 4.99820136696111e-05, "train_wd": 0.04002958960483863, "epoch": 0}
{"train_loss": 9.55839132877896, "train_lr": 0.00014998601063191967, "train_wd": 0.04020716317255338, "epoch": 1}
{"train_loss": 8.00191623267891, "train_lr": 0.00024999000759422837, "train_wd": 0.0405621705635646, "epoch": 2}
{"train_loss": 6.878285548717474, "train_lr": 0.0003499940045565369, "train_wd": 0.04109426142843796, "epoch": 3}
{"train_loss": 6.252864480471249, "train_lr": 0.0004499980015188457, "train_wd": 0.04180291065772983, "epoch": 4}
{"train_loss": 5.574056117297363, "train_lr": 0.0005500019984811545, "train_wd": 0.04268741890020718, "epoch": 5}
{"train_loss": 5.031052095712804, "train_lr": 0.0006500059954434626, "train_wd": 0.043746913253022816, "epoch": 6}
{"train_loss": 4.855693422823692, "train_lr": 0.0007500099924057714, "train_wd": 0.04498034812316445, "epoch": 7}
{"train_loss": 4.6427668024310105, "train_lr": 0.0008500139893680809, "train_wd": 0.046386506259330024, "epoch": 8}
{"train_loss": 4.13732232488126, "train_lr": 0.0009500179863303882, "train_wd": 0.047963999953206556, "epoch": 9}
{"train_loss": 3.831035137641058, "train_lr": 0.0009998995426274404, "train_wd": 0.04971127240897121, "epoch": 10}
{"train_loss": 3.6496348362937154, "train_lr": 0.0009992967042880157, "train_wd": 0.05162659927966087, "epoch": 11}
{"train_loss": 3.6748531173697287, "train_lr": 0.0009980916415549782, "train_wd": 0.0537080903688947, "epoch": 12}
{"train_loss": 4.199905722666321, "train_lr": 0.00099628582261166, "train_wd": 0.055953691496268755, "epoch": 13}
{"train_loss": 4.245502144860612, "train_lr": 0.0009938814475702687, "train_wd": 0.05836118652458501, "epoch": 14}
{"train_loss": 3.9136252777039005, "train_lr": 0.0009908814457914236, "train_wd": 0.060928199546911206, "epoch": 15}
{"train_loss": 3.6716670449570024, "train_lr": 0.0009872894723151784, "train_wd": 0.06365219723131352, "epoch": 16}
{"train_loss": 3.5875498043428315, "train_lr": 0.0009831099034079201, "train_wd": 0.06653049132094936, "epoch": 17}
{"train_loss": 3.7710447650399805, "train_lr": 0.0009783478312305439, "train_wd": 0.06956024128705336, "epoch": 18}
{"train_loss": 4.269760805663826, "train_lr": 0.0009730090576344547, "train_wd": 0.07273845713219447, "epoch": 19}
{"train_loss": 4.632465429347959, "train_lr": 0.0009671000870929053, "train_wd": 0.07606200234104478, "epoch": 20}
{"train_loss": 4.5691923478143295, "train_lr": 0.0009606281187762977, "train_wd": 0.07952759697573956, "epoch": 21}
{"train_loss": 4.460189826863942, "train_lr": 0.0009536010377810942, "train_wd": 0.08313182091278001, "epoch": 22}
{"train_loss": 4.494644553160115, "train_lr": 0.0009460274055230434, "train_wd": 0.08687111721828467, "epoch": 23}
{"train_loss": 4.646373275729964, "train_lr": 0.0009379164493064291, "train_wd": 0.09074179565824961, "epoch": 24}
{"train_loss": 4.8689301321617995, "train_lr": 0.000929278051081998, "train_wd": 0.09474003634036816, "epoch": 25}
{"train_loss": 5.101838638802036, "train_lr": 0.0009201227354073533, "train_wd": 0.09886189348379958, "epoch": 26}
{"train_loss": 5.396307492832676, "train_lr": 0.0009104616566243661, "train_wd": 0.1031032993131822, "epoch": 27}
{"train_loss": 5.59658858689854, "train_lr": 0.000900306585269365, "train_wd": 0.10746006807303266, "epoch": 28}
{"train_loss": 5.702640522488778, "train_lr": 0.0008896698937325205, "train_wd": 0.11192790015858395, "epoch": 29}
```

Would you please let me know if there are any hyperparameters that are different in this vanilla setting except for the number of epochs?

Many thanks!

**Comment by mathildecaron31** [comment]
Hi @fatemeh-slh 

It is expected that when changing the number of epochs the loss values change as well.

If you want to compare apple to apple with one my run at 100 epochs I recommend following the instructions here: 
https://github.com/facebookresearch/dino#vanilla-dino-training-sauropod
Training and evaluation logs are also reported for this vanilla, 100 epochs training.

Hope that helps
